sqlitecluster/SQLite.cpp

```diff
@@ -256,7 +256,8 @@ int SQLite::_walHookCallback(void* sqliteObject, sqlite3* db, const char* name,
 
 void SQLite::_sqliteLogCallback(void* pArg, int iErrCode, const char* zMsg) {
     _mostRecentSQLiteErrorLog = "{SQLITE} Code: "s + to_string(iErrCode) + ", Message: "s + zMsg;
-    SSYSLOG(LOG_INFO, "[info] " << _mostRecentSQLiteErrorLog);
+    SRedactSensitiveValues(_mostRecentSQLiteErrorLog);
+    SINFO(_mostRecentSQLiteErrorLog);
```

Comment:
This seems fine, I'm trying to think of any reason it would have been the way it was, except for an oversight.